src/cheats.c

```diff
@@ -427,6 +423,23 @@ void game_command_cheat(int* eax, int* ebx, int* ecx, int* edx, int* esi, int* e
 	*ebx = 0;
 }
 
+void game_command_unstable_cheat(int* eax, int* ebx, int* ecx, int* edx, int* esi, int* edi, int* ebp)
+{
+	int cheat = *ecx;
```

Comment:
The whole block is to look the same as 'void game_command_cheat' (line 290) which makes me wonder if both should be changed?